docs/use-cases/organizational-access-control.mdx

```diff
@@ -0,0 +1,36 @@
+---
+
+<!--todo: description --->
+
+---
+
+# Organizational Access Control
+
+## Abstract
+
+Typically, organizations employ role-based access control (RBAC) to manage access to restricted resources.
+DAOs often do not have such a mechanism, but both types of organizations suffer from similar problems.
+Some of them are:
+- Members control some entity with the access (e.g their account) and not the access directly;
+- Members reveal not only their roles, but the account as well, violating their privacy;
+- In centralized systems, accounts database may become corrupted leading to access scarceness;
+- Users may not be able to prove the membership in anonymous way.
+
+Even without these problems, members would like to fully control their private data, such as e-mail address, first and last name, telephone number, etc.
+
+![Rarimo social graphs scheme](/img/social-graphs-links-scheme.png)
+
+Similarly, some part of organizations are public in nature, e.g non-profit organizations, and their members often suffer from the publicity of their actions and would be much more comfortable remaining anonymous while being able to prove that they can be trusted.
+
+## Rarimo as a solution
+
+Rarimo's SSI solves these problems and provides many benefits, namely:
+- Members have full control over their data. The user decides whether to disclose their role / name / etc or not.
+- In most cases, the organization itself in some sense will be both an issuer and a verifier, as it issues and then verifies membership credentials.
```

Comment:
Could you clarify your point?

Reply:
I meant that usually there are a lot of issuers/providers, who issue similar credentials (e.g kyc/age credentials etc) and a lot of verifiers who can verify these credentials (and usually verifier != issuer, i.e it's not the same service/person/organisation/etc). Here we have the situation where the *org_name* issues membership credentials and then the same *org_name* will verify them. This is not very important information, but I thought it worth mentioning

Reply:
Let's skip it, it sounds confusing